bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/discovery/DaikinACUnitDiscoveryService.java

```diff
@@ -117,33 +128,54 @@ private Runnable createScanner() {
 
     private boolean receivePacketAndDiscover(DatagramSocket socket) {
         try {
-            // Use a one byte buffer since we don't really care about the contents.
-            byte[] buffer = new byte[1];
+            byte[] buffer = new byte[512];
             DatagramPacket incomingPacket = new DatagramPacket(buffer, buffer.length);
             socket.setSoTimeout(1000 /* one second */);
             socket.receive(incomingPacket);
 
             String host = incomingPacket.getAddress().toString().substring(1);
-            logger.debug("Received packet from {}", host);
+            String data = new String(incomingPacket.getData());
```

Comment:
The default charset varies from system to system so it is better to specify it yourself instead of using the default.

Reply:
done

Reply:
I've changed it to US-ASCII